dev/tests/integration/framework/Magento/TestFramework/Helper/Amqp.php

```diff
@@ -13,26 +13,70 @@
  */
 class Amqp
 {
+    const CONFIG_PATH_HOST = 'queue/amqp/host';
+    const CONFIG_PATH_USER = 'queue/amqp/user';
+    const CONFIG_PATH_PASSWORD = 'queue/amqp/password';
+    const DEFAULT_MANAGEMENT_PORT = '15672';
```

Comment:
What is the reason to load host, user and password from deployment config, but the port from phpunit.xml? I think it's better to store all connection credentials in a single place

Reply:
@sivaschenko Please take a look at this comment above https://github.com/magento/magento2/pull/18978#discussion_r229908654 TLDR: the management plugin port is not needed in production, but is required for the testsuite. I don't think adding it to `env.php` makes sense.

Reply:
Ok, thanks, makes sense.